speedviewlib/src/main/java/com/github/anastr/speedviewlib/RaySpeedometer.kt

```diff
@@ -30,6 +30,7 @@ open class RaySpeedometer @JvmOverloads constructor(
     private var withEffects = true
 
     private var degreeBetweenMark = 5
+    var rayStartSpeed: Float? = null
```

Comment:
The value the ray will be drawn from. This allows the ray to be drawn backwards. null preserves current functionality and draws from startSpeed.